<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgorithmsDialog</class>
 <widget class="QDialog" name="AlgorithmsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Algorytmy</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxSegmentation">
     <property name="title">
      <string>Algorytm segmentacji</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButtonSegm0">
        <property name="text">
         <string>Wzór w RGB</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonSegm1">
        <property name="text">
         <string>Wzór w HSV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonSegm2">
        <property name="text">
         <string>Wzór w YCrCb</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonSegm3">
        <property name="text">
         <string>Historagm skóry twarzy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonSegm4">
        <property name="text">
         <string>Wzór w RGB + histogram skóry twarzy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonSegm5">
        <property name="text">
         <string>Wykrywanie ruchu/eliminacja tła</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxStereo">
     <property name="title">
      <string>Algorytm stereo dopasowania</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonStereo0">
        <property name="text">
         <string>Block Matching (OpenCV)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="bmParamsButton">
        <property name="text">
         <string>Parametry</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButtonStereo1">
        <property name="text">
         <string>Semi-Global Block Matching (OpenCV)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="sgbmParamsButton">
        <property name="text">
         <string>Parametry</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButtonStereo3">
        <property name="text">
         <string>Dopasowanie ROI</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButtonStereo4">
        <property name="text">
         <string>Pixel Matching na ROI</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pixelParamsButton">
        <property name="text">
         <string>Parametry</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="radioButtonStereo5">
        <property name="text">
         <string>FAST Feature Points Matching</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="fastParamsButton">
        <property name="text">
         <string>Parametry</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxLight">
     <property name="title">
      <string>Oświetlenie i tło</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelLight">
        <property name="text">
         <string>Typ oświetlenia jakiegoś</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonGetLight">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Wybierz</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelBkg">
        <property name="text">
         <string>Typ tła jakiegoś</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="buttonGetBkg">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Wybierz</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="manualCheckBox">
     <property name="text">
      <string>manualne oznaczenie początku i końca</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="startStopTrajectoryButton">
     <property name="text">
      <string>Start/stop pobierania trajektorii</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
